examples/run_lm_finetuning.py

```diff
@@ -433,11 +439,16 @@ def main():
     )
     parser.add_argument(
         "--output_dir",
-        default=None,
         type=str,
         required=True,
         help="The output directory where the model predictions and checkpoints will be written.",
     )
+    parser.add_argument(
+        "--model_type", type=str, required=True, help="The model architecture to be trained or fine-tuned.",
+    )
```

Comment:
I passed that option as required as there's no reason to suppose want a BERT model by default anymore (but can revert)